### PR TITLE
Added shimmer mask width

### PR DESCRIPTION
**Pull request by alejandrorosas**
This PR adds the ability to tweak the default value for the shimmer line width.

Keeps compatibility with the old implementation adding the default to 0.5f that is the same in `ShimmerLayout`.

### CODE REVIEW

**Comment by veghtomi** [comment]
@alejandrorosas thanks for the pull request! At the weekend I will check it and merge if no issue found! :)

File: README.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ Following are the attributes and methods to initialise the demo views.
 |```app:shimmer_demo_layout_manager_type``` | ```setDemoLayoutManager(LayoutManagerType)``` | Layout manager of demo view. Can be one among linear_vertical or linear_horizontal or grid. |
 |```app:shimmer_demo_shimmer_color``` | ``` - ``` | Color reference or value. It can be used to change the color of the shimmer line. |
 |```app:shimmer_demo_angle``` | ``` - ``` | Integer value between 0 and 30 which can modify the angle of the shimmer line. The default value is zero. |
+|```app:shimmer_demo_mask_width``` | ``` - ``` | Float value between 0 and 1 which can modify the width of the shimmer line. The default value is 0.5f. |
```
Comment (veghtomi) [inline README.md]: You created a setter method on ShimmerRecyclerView, so for the Java method column the value "-" is not good, replace it with the method name.

Comment (veghtomi) [reply README.md]: Also, for the default value please use 0.5 instead of 0.5f since before you mentioned that it is a Float value.

File: README.md [hunk 2]
```diff
@@ -65,10 +66,10 @@ Define your xml as:
         android:id="@+id/shimmer_recycler_view"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        app:demo_child_count="10"
-        app:demo_grid_child_count="2"
-        app:demo_layout="@layout/layout_demo_grid"
-        app:demo_layout_manager_type="grid"
+        app:shimmer_demo_child_count="10"
```
Comment (veghtomi) [inline README.md:69]: Great job that you have noticed the readme was not updated! Could please add it to a different commit because it is not related to the maskWidth feature?

**Review by veghtomi — CHANGES_REQUESTED** [review]
Please check the messages and fix it according to it!

**Comment by veghtomi** [comment]
@alejandrorosas please check and fix the issues

File: README.md (hunk 2 above)
Comment (alejandrorosas) [reply README.md:69]: Done! Added a new commit for this

**Review by alejandrorosas — COMMENTED** [review]
Addressed review comments :)

**Comment by veghtomi** [comment]
@alejandrorosas nice job!

**Comment by veghtomi** [comment]
@sharish a new release will be required if you have time!

**Comment by sharish** [comment]
@alejandrorosas Thanks for PR and code changes as per review. This will be patched in the next release. @veghtomi Thanks a lot for review and merging!
